exercises/practice/all-your-base/.meta/tests.toml

```diff
@@ -1,69 +1,80 @@
-"a6b476a1-1901-4f2a-92c4-4d91917ae023" = true
-
-# input base is zero
-"e21a693a-7a69-450b-b393-27415c26a016" = true
-
-# input base is negative
-# this test may not be in the suite but has not been intentionally rejected
-# "54a23be5-d99e-41cc-88e0-a650ffe5fcc2" = false
-
-# negative digit
-# this test may not be in the suite but has not been intentionally rejected
-# "9eccf60c-dcc9-407b-95d8-c37b8be56bb6" = false
-
-# invalid positive digit
-"232fa4a5-e761-4939-ba0c-ed046cd0676a" = true
-
-# output base is one
-"14238f95-45da-41dc-95ce-18f860b30ad3" = true
-
-# output base is zero
-"73dac367-da5c-4a37-95fe-c87fad0a4047" = true
-
-# output base is negative
-# this test may not be in the suite but has not been intentionally rejected
-# "13f81f42-ff53-4e24-89d9-37603a48ebd9" = false
-
-# both bases are negative
-# this test may not be in the suite but has not been intentionally rejected
-# "0e6c895d-8a5d-4868-a345-309d094cfe8d" = false
+# This is an auto-generated file. Regular comments will be removed when this
+# file is regenerated. Regenerating will not touch any manually added keys,
+# so comments can be added in a "comment" key.
+
+[5ce422f9-7a4b-4f44-ad29-49c67cb32d2c]
+description = "single bit one to decimal"
+include = true
+
+[0cc3fea8-bb79-46ac-a2ab-5a2c93051033]
+description = "binary to single decimal"
+include = true
+
+[f12db0f9-0d3d-42c2-b3ba-e38cb375a2b8]
+description = "single decimal to binary"
+include = true
+
+[2c45cf54-6da3-4748-9733-5a3c765d925b]
+description = "binary to multiple decimal"
+include = true
+
+[65ddb8b4-8899-4fcc-8618-181b2cf0002d]
+description = "decimal to binary"
+include = true
+
+[8d418419-02a7-4824-8b7a-352d33c6987e]
+description = "trinary to hexadecimal"
+include = true
+
+[d3901c80-8190-41b9-bd86-38d988efa956]
+description = "hexadecimal to trinary"
+include = true
+
+[d68788f7-66dd-43f8-a543-f15b6d233f83]
+description = "empty list"
+include = true
+comments = [
+  "15-bit integer",
+  "this test may not be in the suite but has not been intentionally rejected",
+  "\"5d42f85e-21ad-41bd-b9be-a3e8e4258bbf\" = false"
+]
+
+[5e27e8da-5862-4c5f-b2a9-26c0382b6be7]
+description = "single zero"
+include = true
+
+[2e1c2573-77e4-4b9c-8517-6c56c5bcfdf2]
+description = "multiple zeros"
+include = true
+
+[3530cd9f-8d6d-43f5-bc6e-b30b1db9629b]
+description = "leading zeros"
+include = true
+
+[a6b476a1-1901-4f2a-92c4-4d91917ae023]
+description = "input base is one"
+include = true
+
+[e21a693a-7a69-450b-b393-27415c26a016]
+description = "input base is zero"
+include = true
+
+[232fa4a5-e761-4939-ba0c-ed046cd0676a]
+description = "invalid positive digit"
+include = true
+comments = [
+  "input base is negative",
+  "this test may not be in the suite but has not been intentionally rejected",
+  "\"54a23be5-d99e-41cc-88e0-a650ffe5fcc2\" = false",
+  "negative digit",
+  "this test may not be in the suite but has not been intentionally rejected",
+  "\"9eccf60c-dcc9-407b-95d8-c37b8be56bb6\" = false"
+]
```

Comment:
Looks like the transform script got a bit confused here. IIRC the comment "this test may not be in the suite but has not been intentionally rejected" was itself generated by script, so we should be on the lookout for more instances like that, and potentially fix them by script if there are many.

Reply:
Oh my, terribly sorry! This is due to the fact that rust has commented test cases:

```toml
# negative digit
# this test may not be in the suite but has not been intentionally rejected
# "9eccf60c-dcc9-407b-95d8-c37b8be56bb6" = false
```

I'll see if I can update my script.

Reply:
With `configlet` regenerates the file, these comments will be lost. There are a couple of options:

1. Don't regenerate until you've made a decision what to do with these commented test cases
2. Remove the commented test cases. Functionally, this will cause no change in the behavior of configlet, which will treat them as having an unknown status regardless of whether they are in a comment.
3. Create entries for the commented test cases that have `include = false`, and then add a comment reflecting the status being unknown (`comment = "this test may not be in the suite but has not been intentionally rejected"`).

Reply:
As a practical matter, I think we're going to end up with option 1. The intent for those comments was to simply annotate those test cases for which no decision has yet been made, to make it easier to identify what the decisions are when someone has time to actually review and decide. If configlet's tooling for discovering new test cases is now implemented, option 2 may be better.

Reply:
@coriolinus configlet doesn't yet support the new format. I'll let you know when that is the case. For now, it may be best to make this a draft PR.

Reply:
@coriolinus We've updated configlet to support custom properties. The latest version will also not output `include = true`, as that is the default and thus redundant. 

This PR is not in the best of shapes though, as my script failed to deal with fully commented test cases like:

```toml
# count one word
# this test may not be in the suite but has not been intentionally rejected
# "61559d5f-2cad-48fb-af53-d3973a9ee9ef" = false
```

Seeing how comments are ignored by configlet and will be stripped when re-running configlet, how would you like the above case to be handled?

Reply:
I think the best resolution as far as this PR is just to strip fully-commented test cases entirely. As long as configlet has the capability to provide a list of cases which don't appear in the suite, those comments are now redundant.

Reply:
> As long as configlet has the capability to provide a list of cases which don't appear in the suite, those comments are now redundant.

If you run `./bin/configlet sync --check` (we'll be changing the default to be check soon-ish BTW), it outputs something like:

```
[warn] reverse-string: missing 6 test cases
       - an empty string (c3b7d806-dced-49ee-8543-933fd1719b1c)
       - a word (01ebf55b-bebb-414e-9dec-06f7bb0bee3c)
       - a capitalized word (0f7c07e4-efd1-4aaa-a07a-90b49ce0b746)
       - a sentence with punctuation (71854b9c-f200-4469-9f5c-1e8e5eff5614)
       - a palindrome (1f8ed2f3-56f3-459b-8f3e-6d8d654a1f6c)
       - an even-sized word (b9e7dec1-c6df-40bd-9fa3-cd7ded010c4c)
[warn] wordy: missing 2 test cases
       - multiple multiplication (312d908c-f68f-42c9-aa75-961623cc033f)
       - multiple division (3c854f97-9311-46e8-b574-92b60d17d394)
...
```

Reply:
I've updated the PR to the latest spec.